gaeul/source/org.hwangsaeul.Gaeul2.Source.gschema.xml

```diff
@@ -87,6 +87,9 @@
     <key name="pbkeylen" enum="org.hwangsaeul.Gaeguli.SRTKeyLength">
       <default>"default"</default>
     </key>
+    <key name="use-hw-vaapi-decode" type="b">
```

Comment:
As with the related change in Gaeguli, I suggest shortening the name to "use-vaapi-decode".

Reply:
I am not sure we should specify decode-related parameter for _Source Agent_ because its high-level purpose is stream encoding. That means we need to explain more about this parameter in somewhere why the decoding parameter is required. I feel it is redundant information for user.

Reply:
Removed the redundant informatio.